.automation.conf/tempest/skip-lists/ci-aio-tempest-refstack

```diff
@@ -0,0 +1,8 @@
+tempest.api.compute.servers.test_create_server.ServersTestJSON.test_host_name_is_same_as_server_name: "Ubuntu Noble kernel 6.8.0-57-generic has bug that doesn't work wtih OVS system"
+tempest.api.compute.servers.test_create_server.ServersTestJSON.test_verify_created_server_vcpus: "Ubuntu Noble kernel 6.8.0-57-generic has bug that doesn't work wtih OVS system"
+tempest.api.compute.servers.test_create_server.ServersTestManualDisk.test_host_name_is_same_as_server_name: "Ubuntu Noble kernel 6.8.0-57-generic has bug that doesn't work wtih OVS system"
+tempest.api.compute.servers.test_create_server.ServersTestManualDisk.test_verify_created_server_vcpus: "Ubuntu Noble kernel 6.8.0-57-generic has bug that doesn't work wtih OVS system"
+tempest.api.compute.servers.test_server_actions.ServerActionsTestJSON.test_lock_unlock_server: "Ubuntu Noble kernel 6.8.0-57-generic has bug that doesn't work wtih OVS system"
+tempest.api.compute.servers.test_server_actions.ServerActionsTestJSON.test_reboot_server_hard: "Ubuntu Noble kernel 6.8.0-57-generic has bug that doesn't work wtih OVS system"
+tempest.api.compute.servers.test_server_actions.ServerActionsTestJSON.test_rebuild_server: "Ubuntu Noble kernel 6.8.0-57-generic has bug that doesn't work wtih OVS system"
```

Comment:
Is there a bug report we could track? If so it should be included in the reason the test is skipped.

Reply:
Sure
https://bugs.launchpad.net/ubuntu/+source/linux-meta-oracle-6.8/+bug/2104178

Reply:
Added